PolyEngine/RenderingDevice/OpenGL/Src/GLWorldRendering.cpp

```diff
@@ -82,18 +82,18 @@ void GLRenderingDevice::FillSceneView(SceneView& sceneView)
 		{
 			if (meshCmp->GetBlendingMode() == eBlendingMode::OPAUQE)
 			{
-				sceneView.OpaqueQueue.push(meshCmp);
+				sceneView.OpaqueQueue.emplace(meshCmp);
```

Comment:
When you're not constructing object inside or right before push, don't bother with emplace.
It's actually a decline in performance.